its/autoscan/src/test/resources/autoscan/diffs/diff_S1612.json

```diff
@@ -1,6 +1,6 @@
 {
   "ruleKey": "S1612",
   "hasTruePositives": true,
-  "falseNegatives": 0,
+  "falseNegatives": 1,
```

Comment:
I remember you asked a question about this one.
It is a valid change, as you need a test case with lambdas.